src/stores/ui.store.ts

```diff
@@ -243,6 +243,7 @@ end tell`)
     weatherApiKey: '' as string,
     weatherLat: '' as string,
     weatherLon: '' as string,
+    weatherLang: '' as string,
```

Comment:
it's better if you add a global setting

Reply:
you mean something like this: `weatherLang: 'en' as string`

Reply:
yeah, you will need some UI for setting the language as well as a proper list of supported languages and automatic detection of the OS locale/language

Reply:
as a matter of fact, there needs to be a whole Settings UI. It's a lot of work and that's why I haven't tackled this issue so far... but if this is something you really want I won't really merge this kind of temporary solution

Reply:
Now let's do this for the moment I close the pull request.

At the moment I think you should fix the following problems:

1) Multilingual support
After introducing this you will decide how to organize this, whether with the automatic detection of the locale / language of the operating system or other

2) What to put in the whole system interface whether to insert this too or not

3) I tried to use `@react-native-picker/picker` to make languages select for bees, but I couldn't get it to work on macos, weird errors.

P.s.
Take a look at this graphical lib for input fields and chips and much more it would be nice to be able to use them in your project.

In time these are the languages supported by the api system.

```json
[
  {"af": "Afrikaans"},
  {"al": "Albanian"},
  {"ar": "Arabic"},
  {"az": "Azerbaijani"},
  {"bg": "Bulgarian"},
  {"ca": "Catalan"},
  {"cz": "Czech"},
  {"da": "Danish"},
  {"de": "German"},
  {"el": "Greek"},
  {"en": "English"},
  {"eu": "Basque"},
  {"fa": "Persian (Farsi)"},
  {"fi": "Finnish"},
  {"fr": "French"},
  {"gl": "Galician"},
  {"he": "Hebrew"},
  {"hi": "Hindi"},
  {"hr": "Croatian"},
  {"hu": "Hungarian"},
  {"id": "Indonesian"},
  {"it": "Italian"},
  {"ja": "Japanese"},
  {"kr": "Korean"},
  {"la": "Latvian"},
  {"lt": "Lithuanian"},
  {"mk": "Macedonian"},
  {"no": "Norwegian"},
  {"nl": "Dutch"},
  {"pl": "Polish"},
  {"pt": "Portuguese"},
  {"pt_br": "Português Brasil"},
  {"ro": "Romanian"},
  {"ru": "Russian"},
  {"sv": "Swedish"},
  {"sk": "Slovak"},
  {"sl": "Slovenian"},
  {"es": "Spanish"},
  {"sr": "Serbian"},
  {"th": "Thai"},
  {"tr": "Turkish"},
  {"uk": "Ukrainian"},
  {"vi": "Vietnamese"},
  {"zh_cn": "Chinese Simplified"},
  {"zh_tw": "Chinese Traditional"},
  {"zu": "Zulu"}
]
```

Reply:
I added myself support for macos on react-native-picker, so it should be working, but I don't like it, the app is not really using macOS native components

As for the languages, sure that list looks OK, you can take a look into what I created for raycast with the google translate plugin, there is a more complete list with flags in there